backend_services_deployment.md

```diff
@@ -12,6 +12,8 @@ podified OpenStack control plane services.
 * The `openstack-operator` deployed, but `OpenStackControlPlane`
   **not** deployed.
 
+* The MariaDB PVC is bound.
```

Comment:
I don't think that's correct. We need an available PV to start with, but the PVC shouldn't be bound yet. The PVC will be created by the MariaDB deployment, not by `make crc_storage`. I would formulate the prerequisite as:

* Available free PVs that can be claimed by MariaDB and RabbitMQ. (In developer environments driven by install_yamls, run `make crc_storage`.)